app/views/supplementary/package_information.scala.html

```diff
@@ -19,19 +19,51 @@
 @import forms.supplementary.PackageInformation
 @import uk.gov.hmrc.play.views.html._
 
-@(appConfig: AppConfig, form: Form[PackageInformation])(implicit request: Request[_], messages: Messages)
+@(form: Form[_], packages: Seq[PackageInformation])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@tableHeader = @{
+if(packages.size == 1) {
+messages("supplementary.packageInformation.table.heading")
+} else {
+messages("supplementary.packageInformation.table.multiple.heading", packages.size)
+}
+}
 
 @main_template(
     title = messages("supplementary.packageInformation.title"),
     appConfig = appConfig
 ) {
-    @helpers.form(PackageInformationController.submit(), 'autoComplete -> "off") {
+    @helpers.form(PackageInformationController.submitForm(), 'autoComplete -> "off") {
         @components.back_link("/customs-declare-exports/declaration/supplementary/item-type")
 
         @components.error_summary(form.errors)
 
         @components.page_title(Some("supplementary.packageInformation.title"))
 
+        @if(packages.nonEmpty) {
+            <table class="form-group">
+                <caption>@tableHeader</caption>
+                <thead>
+                <tr>
+                    <th scope="col">@messages("supplementary.packageInformation.typesOfPackages")</th>
+                    <th scope="col">@messages("supplementary.packageInformation.numberOfPackages")</th>
+                    <th scope="col">@messages("supplementary.packageInformation.shippingMarks")</th>
+                    <th scope="col">@messages("supplementary.packageInformation.remove")</th>
+                </tr>
+                </thead>
+                <tbody>
+                @packages.zipWithIndex.map { case (item, index) =>
```

Comment:
:+1: We should use this approach from now on

Reply:
this needs to be moved into a component so every screen can use, we will add a tech task to do that

Reply:
Yes please!!!! :heart: